### PR TITLE
feat(select): allow on Blur

### PR DESCRIPTION
**Pull request by juanigalan91**
# General summary
Added the possibility to detect whether the `Select` component was blurred or not:
- Since the Select is actually not an input, we need to fake the `blur` with a click away
- Updated the main story since it did not change the option selected upon clicking the option

# Check list

-   [ ] (if has style) Add `need: review-integration` label
-   [ ] (if has textual documentation) Add `need: review-doc` label
-   [X] (if has code) Add `need: review-frontend` label
-   [ ] (if has react) Check through the [react dev check list]
-   [ ] (if fix or feature) Add `need: test` label

Check out the [contribution guide] for general guidelines for submissions to the design system.

[contribution guide]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md
[react dev check list]: https://github.com/lumapps/design-system/blob/master/CONTRIBUTING.md#react-developpment-check-list

### CODE REVIEW

File: packages/lumx-react/src/components/select/Select.tsx [hunk 1]
```diff
@@ -249,13 +268,47 @@ const Select: React.FC<SelectProps> = ({
...
+        setWasBlurred(false);
+    };
+
     useFocus(anchorRef.current, Boolean(isOpen));
-    useHandleElementFocus(anchorRef.current, setIsFocus);
+    useHandleElementFocus(anchorRef.current, setIsFocus, onSelectFocus);
+
+    /**
+     * Since the select is not really an input, we need to fake the onBlur event.
+     * We can do this by listening to the click away event, which is basically an "onBlur"
```
Comment (pidupuis) [inline packages/lumx-react/src/components/select/Select.tsx]: Smart

**Comment by matmkian** [comment]
Test ko, need to demonstrate but when dropdown is open, if you click on another input, blur is triggered on second click.

**Comment by juanigalan91** [comment]
@matmkian seems that this is an error that we have in master as well, since upon clicking on another input, the select is still focused.

In any case, I will take a look at the bug!

**Comment by matmkian** [comment]
@juanigalan91 you're right, I just checked :-) Thank you!

**Comment by matmkian** [comment]
@juanigalan91 Also, onBlur event is not triggered if you leave the field with tab key down.

**Pull request merged**